.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+indent_size = 2
```

Comment:
I don't think anyone will ever modify the LICENSE file, .gitignore file shouldn't include any indentation. 
For the README, it depends on which indent size you think it should be set (for now we don't use it anyway)